### PR TITLE
Seed/unseed

### PR DESCRIPTION
**Pull request by jc121F1**
Add Seed/Unseed commands, and club utility methods for adding and deleting users.

### CODE REVIEW

File: BookClub/tests/models/test_add_user_club_model.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            self.club1.full_clean()
+
+    def test_valid_club(self):
+        self._assert_club_is_valid()
+        
+    def testAddUserValidInfo(self):
+        membershipBefore = ClubMembership.objects.filter(club=self.club1).count()
+        self.club1.add_user(self.testUser, ClubMembership.UserRoles.MEMBER)
+        membershipAfter = ClubMembership.objects.filter(club=self.club1).count()
+        self.assertLess(membershipBefore, membershipAfter)
```
Comment (clerav) [inline BookClub/tests/models/test_add_user_club_model.py:44]: For all tests using assertLess(), it's better to use assertEquals with before + 1 for better accuracy

File: BookClub/tests/models/test_add_user_club_model.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        membershipBefore = ClubMembership.objects.filter(club=self.club1).count()
+        self.club1.add_owner(self.testUser)
+        membershipAfter = ClubMembership.objects.filter(club=self.club1).count()
+        self.assertEqual(membershipBefore, membershipAfter)
+        
+    def testAddOwnerNoOwnerExists(self):
+        testClub = Club.objects.create(name="Test",description="testclub",rules="test",tagline="tagline", is_private=False)
+        membershipBefore = ClubMembership.objects.filter(club=testClub).count()
+        testClub.add_owner(self.testUser)
+        membershipAfter = ClubMembership.objects.filter(club=testClub).count()
```
Comment (clerav) [inline BookClub/tests/models/test_add_user_club_model.py:72]: use additional filter membership = ClubMembership.UserRoles.OWNER for accuracy

Comment (clerav) [reply BookClub/tests/models/test_add_user_club_model.py:72]: nvm, this is already done

File: BookClub/tests/models/test_add_user_club_model.py (hunk 1 above)
Comment (jc121F1) [reply BookClub/tests/models/test_add_user_club_model.py]: Solved

**Review by amir-rahim — APPROVED** [review]

**Pull request merged**